drone/values.yaml

```diff
@@ -0,0 +1,66 @@
+image:
+  registry: docker.io
+  org: drone
+  name: drone
+  tag: 0.7.3
+  pullPolicy: IfNotPresent
+
+service:
+  http:
+    externalPort: 80
+    internalPort: 8000
+  type: NodePort
+
+ingress:
+  enabled: true
+  # enable TLS via kube-lego
+  tls: false
+  hostname: drone2.srcd.host
+  annotations:
+    kubernetes.io/ingress.class: gce
+    kubernetes.io/ingress.global-static-ip-name: drone2-srcd-host
+
+# Provide with 'helm install', and do NOT change it when doing 'helm upgrade'
+# shared_secret: ""
+
+server:
+  db_path: "/var/lib/drone/"
+  db_disk: "data-prod-drone"
+  # Drone server configuration. Values in here get injected as environment variables.
+  # See http://readme.drone.io/admin/installation-reference#server-options for a list of possible values.
+  env:
+    DRONE_DEBUG: "true"
+    DRONE_BROKER_DEBUG: "true"
+    DRONE_DATABASE_DRIVER: "sqlite3"
+    DRONE_DATABASE_DATASOURCE: "/var/lib/drone/drone.sqlite"
+    # Drone requires some environment variables to bootstrap the git service or it won't start up.
+    # Uncomment this and add your own custom configuration.
+    #
+    # See http://readme.drone.io/admin/installation-reference/ for more info on these envvars.
+    DRONE_OPEN: "true"
+    DRONE_GITHUB: "true"
+    DRONE_ADMIN: "mcuadros,rporres,smeruelo"
```

Comment:
Sure, but I have not been able to do it manually (some issue parsing the commas), I hope it just works programmatically from the drone pipeline or will check later how to fix it.

Anyway, I just thought it could be an inoffensive default value (like the external disk name, for instance).